<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NI9234</class>
 <widget class="QWidget" name="NI9234">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NI9234</string>
  </property>
  <widget class="QGroupBox" name="ControlButtons_GroupBox">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QPushButton" name="Start_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateTask_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearTask_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Task</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Parameters_Label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parameters!!&lt;/p&gt;&lt;p&gt;Will show at here!!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>371</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="SpectrumVBoxLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QGraphicsView" name="SpectrumChannel0_GraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="SpectrumChannel1_GraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="SpectrumChannel2_GgraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="SpectrumChannel3_GgraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>381</width>
     <height>701</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="WaveVBoxLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QGraphicsView" name="WaveChannel0_GraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="WaveChannel1_GraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="WaveChannel2_GraphicsView"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="WaveChannel3_GraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="WriteFile_GroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Write File</string>
   </property>
   <widget class="QCheckBox" name="WriteFile_CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WriteFile_LineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WriteFile_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Directory</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="WriteFileType_ComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="NowTime_Label">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>690</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Now Time</string>
   </property>
  </widget>
  <widget class="QFrame" name="PreparationSetting_Frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>361</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="DAQParameters_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>340</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>DAQ Parameters</string>
    </property>
    <widget class="QFrame" name="SampleRate_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QSlider" name="SampleRate_HorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="SampleRate_SpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="specialValueText">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="SampleRate_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sampling Rate (Hz)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="FrameDuration_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QSlider" name="FrameDuration_HorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="FrameDuration_SpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="specialValueText">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="FrameDuration_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Frame Duration (ms)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="BufferDuration_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QSlider" name="BufferDuration_HorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="BufferDuration_SpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="specialValueText">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="BufferDuration_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Buffer Duration (ms)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ChannelOption_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>210</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Channel Option</string>
    </property>
    <widget class="QFrame" name="Channel0_Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>190</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QCheckBox" name="Channel0_CheckBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel 0</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Channel0_ComboBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>5</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Channel2_Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>190</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QCheckBox" name="Channel2_CheckBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel 2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Channel2_ComboBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>5</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Channel3_Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>190</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QCheckBox" name="Channel3_CheckBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel 3</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Channel3_ComboBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>5</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Channel1_Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>190</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QCheckBox" name="Channel1_CheckBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>75</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel 1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Channel1_ComboBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>5</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GraphParameters_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph Parameters</string>
    </property>
    <widget class="QFrame" name="DownSample_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QSlider" name="DownSample_HorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="DownSample_SpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="specialValueText">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="DownSample_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Graph Down Sample</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="UpdateInterval_Frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QSlider" name="UpdateInterval_HorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>5</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="UpdateInterval_SpinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="specialValueText">
       <string notr="true"/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="UpdateInterval_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Interval (ms)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Task_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Task</string>
    </property>
    <widget class="QLabel" name="TaskName_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Task Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TaskName_LineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SampleRate_HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SampleRate_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SampleRate_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SampleRate_HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FrameDuration_HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameDuration_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FrameDuration_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FrameDuration_HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpdateInterval_HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UpdateInterval_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpdateInterval_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>UpdateInterval_HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSample_HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DownSample_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownSample_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DownSample_HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BufferDuration_HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BufferDuration_SpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BufferDuration_SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BufferDuration_HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
